app/controllers/submissions_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class SubmissionsController < ApplicationController
   include SubmissionsHelper
+  include ApplicationHelper
```

Comment:
@jeremygoh Actually, can you try without the `ApplicationHelper` as well? I seem to recall it's already included in ApplicationController.